### DISCUSSION TITLE
destroyed: no one is publishing to path

### DISCUSSION THREAD

**Discussion by cooperbrown9** [bluenviron/mediamtx#3309] [Questions]
### Question

I am running the media server and am constantly spinning up and down some custom source streams. `-e MTX_PATHS_${STREAM_NAME}_SOURCE=rtsp://.....`
I notice when I shut down my media server and spin it back up, it will try to hit old paths that are no longer in use. For instance, I might prepend everything with `ABC-`, then shut everything down. Then the next spinup I will prepend it with `XYZ`, but I am still getting these  `no one is publish to path "ABC-source1"` logs. 

Does anybody know how to clear out the unused paths in MediaMTX? I'm afraid it's affecting performance

<img width="601" alt="Screenshot 2024-05-01 at 9 33 36 AM" src="https://github.com/bluenviron/mediamtx/assets/12388607/a00cecea-052d-4be2-9af1-56e71ad28ac2">

**Comment by aler9** [comment]
Hello, the entity that is trying to hit old paths is not the server itself, but web clients that are reading the stream through HLS. If the server is shut down and there are pending clients, these will try to reconnect to their desired path, resulting in the error "no one is publishing to path XX". No paths are automatically created and no cleanup is needed.

[a comment by an automated account is not shown]
